index.html

```diff
@@ -4585,7 +4617,10 @@ <h5>Conclusion</h5>
 See examples.
 -->
 <p>
-Based on the limited data, we are <a data-ucr-role="conclusion">undecided</a> about whether this should be a requirement.
+It might seem that, as controls are implemented using existing interactive HTML elements,
+there is no pressing need for them to be supported by additional APIS.
```

Comment:
s/APIS/APIs

Reply:
I've bumped up the highlighting of typos in my IDE in the hope I catch more of these before committing! Thanks for spotting them :-)